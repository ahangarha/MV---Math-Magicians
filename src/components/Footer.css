footer {
  text-align: center;
  padding: 2rem 1rem;
  background-color: #000;
}

footer a {
  margin-inline: 0.25rem;
}
