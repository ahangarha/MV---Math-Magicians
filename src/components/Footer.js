import React from 'react';
import './Footer.css';

export default function Footer() {
  return (
    <footer>
      This is a Free/Libre Software under&nbsp;
      <a
        href="https://github.com/ahangarha/MV-MathMagicians/"
        target="_blank"
        rel="noopener noreferrer"
      >
        AGPL license
      </a>
      .
    </footer>
  );
}
