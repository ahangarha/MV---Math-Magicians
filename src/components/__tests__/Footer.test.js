import React from 'react';
import Renderer from 'react-test-renderer';
import Footer from '../Footer';

it('Renders correctly', () => {
  const tree = Renderer
    .create(<Footer />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
