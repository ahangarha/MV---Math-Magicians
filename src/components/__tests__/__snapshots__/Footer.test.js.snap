// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<footer>
  This is a Free/Libre Software underÂ 
  <a
    href="https://github.com/ahangarha/MV-MathMagicians/"
    rel="noopener noreferrer"
    target="_blank"
  >
    AGPL license
  </a>
  .
</footer>
`;
